activesupport/test/logger_test.rb

```diff
@@ -16,6 +16,21 @@ def setup
     @logger  = Logger.new(@output)
   end
 
+  def with_level(level = Logger::INFO)
```

Comment:
It doesn't really make sense with a default argument here. What does `with_level(nil)` mean? Let's go explicit.